src/cli/main/__main__.py

```diff
@@ -134,9 +134,10 @@ def _nix_build(
     src: str,
 ) -> List[str]:
     if cache is None:
-        substituters = "https://cache.nixos.org"
+        substituters = "https://cache.nixos.org https://hydra.iohk.io"
         trusted_pub_keys = (
-            "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY="
+            "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= "
+            "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= "
```

Comment:
This is used only for the first builds (getting the list of attrs + a json with the cache config) when Makes starts, so cache.nixos.org is enough as we only fetch a few things from nixpkgs

Reply:
ooops sry...

Reply:
sorry, this was a stupid rebasing / merging error because of the way I tested this PR (for which I better had that substituter set).